test_communication/CMakeLists.txt

```diff
@@ -24,35 +21,27 @@ ament_auto_find_build_dependencies()
 
 option(SECURITY "Activate security" OFF)
 
-set(message_files
-  "msg/BoundedArrayNested.msg"
-  "msg/BoundedArrayPrimitives.msg"
-  "msg/Builtins.msg"
-  "msg/DynamicArrayNested.msg"
-  "msg/DynamicArrayPrimitives.msg"
-  "msg/DynamicArrayPrimitivesNested.msg"
-  "msg/Empty.msg"
-  "msg/Nested.msg"
-  "msg/Primitives.msg"
-  "msg/StaticArrayNested.msg"
-  "msg/StaticArrayPrimitives.msg"
-)
-set(other_message_files
-  "msg/FieldsWithSameType.msg"
-  "msg/UInt32.msg"
-)
-set(service_files
-  "srv/Empty.srv"
-  "srv/Primitives.srv"
-)
-rosidl_generate_interfaces(${PROJECT_NAME}
-  ${message_files}
-  ${other_message_files}
-  ${service_files}
-  DEPENDENCIES builtin_interfaces
-)
-
 if(BUILD_TESTING)
+
```

Comment:
Nitpick: no empty line after `if`.

Reply:
fixed in https://github.com/ros2/system_tests/pull/223/commits/db21f8b81f1a3255c09c74ab7a7f57d5e255d310